package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "aptos",
   "description": "Aptos SDK",
-  "packageManager": "pnpm@8.3.1",
+  "packageManager": "pnpm@8.9.0",
```

Comment:
why?

Reply:
Lockfiles were giving me a hard time, I found some bug that is fixed in like 8.7+. Any reason not to?

Reply:
nope, just curios